tests_integration/test_temporary_connection.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+
+@pytest.mark.integration
+def test_temporary_connection(runner):
+    result = runner._invoke(
```

Comment:
Can we test a successful connection?

Maybe we can run CLI with an empty config (`--config-file` flag used in `invoke_with_config`) (additionally confirming that `dev` connection is not needed) and with `--temporary-connection` flag?

Reply:
And the values of flags we can get from env variables. WDYT?

Reply:
I was talking about it with @sfc-gh-astus, and we decided there is a risk of a password leak